### PR TITLE
Updates rack-protection

### PR DESCRIPTION
**Pull request by omgitsbillryan**
## Description of change
Updates `rack-protection`.

Command I ran:
```
bundle update --conservative rack-protection
```

## Testing done
Local rspec.

## Testing planned
None.

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by omgitsbillryan** [comment]
Investigation as to the safety of this update was done in this PR: https://github.com/department-of-veterans-affairs/vets-api/pull/2427

TL;DR - it appears to be a safe update
